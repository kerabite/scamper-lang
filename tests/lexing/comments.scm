(define ; a line comment
  foo (lambda;another line comment)
    (x #| a block comment
    
    
|#) (+; more comments
#|y|#x;42
1));hah
)

(foo 10)