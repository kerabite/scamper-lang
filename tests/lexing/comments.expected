(: (1, 1) => (1, 1)
define: (1, 2) => (1, 7)
foo: (2, 3) => (2, 5)
(: (2, 7) => (2, 7)
lambda: (2, 8) => (2, 13)
(: (3, 5) => (3, 5)
x: (3, 6) => (3, 6)
): (6, 3) => (6, 3)
(: (6, 5) => (6, 5)
+: (6, 6) => (6, 6)
x: (7, 6) => (7, 6)
1: (8, 1) => (8, 1)
): (8, 2) => (8, 2)
): (8, 3) => (8, 3)
): (9, 1) => (9, 1)
(: (11, 1) => (11, 1)
foo: (11, 2) => (11, 4)
10: (11, 6) => (11, 7)
): (11, 8) => (11, 8)
